siliconcompiler/schema/schema_cfg.py

```diff
@@ -141,7 +139,20 @@ def schema_cfg():
             schelp="""Name of the top level module or library. Required for all
             chip objects.""")
 
+    # Lambda value
+    scparam(cfg,['lambda'],
```

Comment:
Does this belong in the top-level schema? It seems like it might only apply to a specific type of PDK, if I'm reading the description correctly.